tests/cypress/specs/feature/eligibility.cy.js

```diff
@@ -1,49 +1,40 @@
 const agencies = require("../../fixtures/transit-agencies");
 const users = require("../../fixtures/users.json");
 
-const eligibility_url = "/eligibility/confirm";
-
 describe("Eligibility confirmation flow", () => {
```

Comment:
Wasn't sure if we want to explicitly change this to "Courtesy Cards" or what.

Reply:
Let's do that, so it's more clear

Reply:
Added that to https://github.com/cal-itp/benefits/pull/1277, since I'm moving files around there.